sparse/src/KokkosSparse_CcsMatrix.hpp

```diff
@@ -229,23 +229,27 @@ class CcsMatrix {
   }
 
   //! The number of rows in the sparse matrix.
-  KOKKOS_INLINE_FUNCTION ordinal_type numCols() const {
+  KOKKOS_INLINE_FUNCTION ordinal_type ::numCols() const {
```

Comment:
Why do these member functions need to be declared like this? The ``::name`` syntax is for referencing things in the global namespace, right?

Reply:
With my last force push, this should be handled properly now.